### PR TITLE
doc(requirements): Add management group for multiple subscriptions

### PR DESCRIPTION
**Pull request by puchy22**
### Context

Now in documentation we recommend users assign permissions subscription by subscription. Would be better to assign permissions to a management group and inside of this management group add all subscriptions to audit.

### Description

New documentation well written.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/getting-started/requirements.md [hunk 1]
```diff
@@ -114,6 +114,24 @@ To assign this roles, follow the instructions:
 
 *Repeat these steps for `Reader` role*
 
+Moreover, some additional read-only permissions are needed for some checks, for this kind of checks that are not covered by built-in roles we use a custom role. This role is defined in [prowler-azure-custom-role](https://github.com/prowler-cloud/prowler/blob/master/permissions/prowler-azure-custom-role.json). Please be sure to change the `assignableScopes` field for your subscriptions or management group. Once the cusotm role is created, repeat the steps mentioned above to assign the new `ProwlerRole` to an identity.
+
+##### Recommendation for multiple subscription
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
##### Recommendation for multiple subscriptions
```

File: docs/getting-started/requirements.md [hunk 2]
```diff
@@ -114,6 +114,24 @@ To assign this roles, follow the instructions:
...
+
+For scanning multiple subscription it could be tedious create and assign role for every subscription, for this reason in Prowler recommend the usage of *[management groups](https://learn.microsoft.com/en-us/azure/governance/management-groups/overview)* to group all subscriptions that wanna be audited.
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
While scanning multiple subscriptions could be tedious to create and assign roles for each one. For this reason in Prowler we recommend the usage of *[management groups](https://learn.microsoft.com/en-us/azure/governance/management-groups/overview)* to group all subscriptions that are going to be audited by Prowler.
```

File: docs/getting-started/requirements.md [hunk 3]
```diff
@@ -114,6 +114,24 @@ To assign this roles, follow the instructions:
...
+
+To do this in a propper way you have to [create a new management group](https://learn.microsoft.com/en-us/azure/governance/management-groups/create-management-group-portal) and add all roles in the same way that have been done for subscription scope.
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
To do this in a propper way you have to [create a new management group](https://learn.microsoft.com/en-us/azure/governance/management-groups/create-management-group-portal) and add all the roles in the same way that have been done for subscription scope.
```

File: docs/getting-started/requirements.md [hunk 4]
```diff
@@ -114,6 +114,24 @@ To assign this roles, follow the instructions:
...
+
+![Create management group](../img/create-management-group.gif)
+
+Once the management group is properly set you can add all the subscription that you want to audit.
+
+![Add subscription to management group](../img/add-sub-to-management-group.gif)
+
+???+ note
+    By default, `prowler` will scan all subscriptions in the Azure tenant, use flag `--subscription-id` to specify the subscriptions to be scanned.
```
Comment (jfagoagas) [inline docs/getting-started/requirements.md]:
```suggestion
    By default, `prowler` will scan all subscriptions in the Azure tenant, use the flag `--subscription-id` to specify the subscriptions to be scanned.
```

**Review by jfagoagas — DISMISSED** [review]
Great work @puchy22 👏 Always taking care of the documentation 🏅

File: docs/tutorials/azure/subscriptions.md [hunk 5]
```diff
@@ -1,10 +1,47 @@
 # Azure subscriptions scope
 
-By default, Prowler is multisubscription, which means that is going to scan all the subscriptions is able to list. If you only assign permissions to one subscription, it is going to scan a single one.  
+By default, Prowler is multisubscription, which means that is going to scan all the subscriptions is able to list. If you only assign permissions to one subscription, it is going to scan a single one.
```
Comment (pedrooot) [inline docs/tutorials/azure/subscriptions.md:3]:
```suggestion
By default, Prowler is multisubscription, which means that is going to scan all the subscriptions is able to list. If you only assign permissions to one subscription, it's going to scan a single one.
```

**Review by pedrooot — APPROVED** [review]
Nice job! @puchy22 🔝

**Pull request merged**